libs/core/src/components.d.ts

```diff
@@ -350,6 +350,102 @@ export namespace Components {
          */
         "value": string;
     }
+    interface PdsSelect {
+        /**
+          * A unique identifier for the combobox
```

Comment:
Nitpick, but can periods be added to the descriptions?

Reply:
Even if it's not a sentence?

Reply:
Please see the other components since the ID description was recently updated across the board. As far as a sentence or not, I think we've been adding periods across the board.

Reply:
So, just to be clear, every description of every prop should have a period at the end, yes?

Reply:
I mean unless there's something I am missing I don't see why not? It can also be done in a follow-up since I didn't check them all across the board.